src/components/MDXComponents.tsx

```diff
@@ -1,37 +1,16 @@
-import { Heading, Link, Stack, Text } from '@chakra-ui/react';
+import { Flex, Heading, Link, Stack, Table, Text } from '@chakra-ui/react';
 import NextLink from 'next/link';
-import { PrismLight as SyntaxHighlighter } from 'react-syntax-highlighter';
 
-import { Code } from './UI/docs'
+import { Code } from './UI/docs';
+import { textStyles } from '../theme/foundations';
 
-import bash from 'react-syntax-highlighter/dist/cjs/languages/prism/bash';
-import go from 'react-syntax-highlighter/dist/cjs/languages/prism/go';
-import graphql from 'react-syntax-highlighter/dist/cjs/languages/prism/graphql';
-import java from 'react-syntax-highlighter/dist/cjs/languages/prism/java';
-import javascript from 'react-syntax-highlighter/dist/cjs/languages/prism/javascript';
-import json from 'react-syntax-highlighter/dist/cjs/languages/prism/json';
-import python from 'react-syntax-highlighter/dist/cjs/languages/prism/python';
-import sh from 'react-syntax-highlighter/dist/cjs/languages/prism/shell-session';
-import solidity from 'react-syntax-highlighter/dist/cjs/languages/prism/solidity';
-import swift from 'react-syntax-highlighter/dist/cjs/languages/prism/swift';
-
-// syntax highlighting languages supported
-SyntaxHighlighter.registerLanguage('bash', bash);
-SyntaxHighlighter.registerLanguage('go', go);
-SyntaxHighlighter.registerLanguage('graphql', graphql);
-SyntaxHighlighter.registerLanguage('java', java);
-SyntaxHighlighter.registerLanguage('javascript', javascript);
-SyntaxHighlighter.registerLanguage('json', json);
-SyntaxHighlighter.registerLanguage('python', python);
-SyntaxHighlighter.registerLanguage('sh', sh);
-SyntaxHighlighter.registerLanguage('solidity', solidity);
-SyntaxHighlighter.registerLanguage('swift', swift);
+const { header1, header2, header3, header4 } = textStyles;
 
 const MDXComponents = {
   // paragraphs
   p: ({ children }: any) => {
     return (
-      <Text mb={7} _last={{ mb: 0 }} size='sm' lineHeight={1.5}>
+      <Text mb={7} size='sm' lineHeight={1.5}>
```

Comment:
text size here [should be 16px](https://www.figma.com/file/ekzIgwyeVKLtFSAcnA0Q0D/geth-website?node-id=694%3A2580&t=hfe4htIHQvUUNT87-0), defined on theme

Reply:
It's rendering at 16px with or without `size='sm'`... will remove.